propka/bonds.py

```diff
@@ -370,16 +368,17 @@ def put_atom_in_box(self, x, y, z, atom):
             z:  box z-coordinates
             atom:  the atom to place in a box
         """
-        for bx in [x, x+1]:
-            for by in [y, y+1]:
-                for bz in [z, z+1]:
-                    key = (bx, by, bz)
+        for box_x in [x, x+1]:
+            for box_y in [y, y+1]:
+                for box_z in [z, z+1]:
+                    key = (box_x, box_y, box_z)
                     try:
                         self.boxes[key].append(atom)
                     except KeyError:
                         pass
 
-    def has_bond(self, atom1, atom2):
+    @classmethod
```

Comment:
could even be a `staticmethod`

Reply:
Addressed in https://github.com/Electrostatics/propka-3.1/commit/db8571414e01b7451790c54450f90aa875cbd180